portfolio/src/main/webapp/sheetsApi.js

```diff
@@ -113,39 +118,110 @@ function handleAuthorizationError(error) {
  * message on the UI.
  */
 function handleExportError(reason) {
-  var $sheetsMessage = document.getElementById('sheets-message');
-  $sheetsMessage.removeClass('d-none')
+  $('#sheets-message')
+      .removeClass('d-none')
       .text('Error: ' + reason.result.error.message)
       .show();
 }
 
+/**
+ * Helps the SheetsApi requests easily create the values it needs for a
+ * spreadsheet.
+ */
+function makeSheetsValuesFromScheduledTasks(scheduledTasks, values) {
+  for (index = 0; index < scheduledTasks.length; index++) {
```

Comment:
I do not know how to make this work with a forEach() suggestions are appreciated.

Reply:
I think for loop is fine. But maybe you can use var index = 0 rather than just index = 0

Reply:
Done.